website/docs/d/network_ddos_protection_plan.html.markdown

```diff
@@ -0,0 +1,47 @@
+---
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_network_ddos_protection_plan"
+sidebar_current: "docs-azurerm-datasource-network-ddos-protection-plan-x"
+description: |-
+  Use this data source to access information about an existing Azure Network DDoS Protection Plan.
+
+---
+
+# Data Source: azurerm_network_ddos_protection_plan
+
+Manages an AzureNetwork DDoS Protection Plan.
+
+-> **NOTE** Azure only allow `one` DDoS Protection Plan per region.
```

Comment:
This doesn't make sense for a data source?

Reply:
I knew I forgot to take out something! 🤦‍♀